### DISCUSSION TITLE
How to set keymap for Sudawrite?

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#3049] [Q&A]
Ctrl + Shift + s to trigger :SudaWrite<CR>?

**Comment by dpetka2001** [comment]
Use `vim.keymap.set` to define the keymap you want in file `/lua/config/keymaps.lua`. You should also make sure that the keymap you want to use doesn't conflict with some mapping in the terminal you use.

**Reply by tonmoy998** [reply]
I dont know why it is not not working 
local opts = { noremap = true, silent = true }

local term_opts = { silent = true }

-- Shorten function name
local keymap = vim.api.nvim_set_keymap
keymap("", "<Space>", "<Nop>", opts)

keymap("n", "<C-S>", ":SudaWrite<CR>", opts)

**Reply by dpetka2001** [reply]
What is the name of the plugin you're using, so I can try it?

**Reply by tonmoy998** [reply]
here is the plugin: https://github.com/lambdalisue/suda.vim

**Reply by dpetka2001** [reply]
Try the following in a file in `~/.config/nvim/lua/plugins/suda.lua`
```lua
return {
  {
    "lambdalisue/suda.vim",
    init = function()
      vim.keymap.set("n", "<leader>sA", "<cmd>SudaWrite<cr>", { desc = "SudaWrite" })
    end,
  },
}
```
Change `<leader>sA` to whatever you want, as long as it doesn't conflict with mappings for your terminal.
